features/frameworks/mbed-trace/MbedTraceInitialiser.cpp

```diff
@@ -0,0 +1,12 @@
+#include "mbed_trace.h"
+
+class MbedTraceInitialiser {
+public:
+    MbedTraceInitialiser() {
```

Comment:
only ctor? no deinit?

Reply:
there could ve ctor - but would we benefit of it somehow?